setup.py

```diff
@@ -27,7 +27,7 @@
     install_requires=[
         "attrdict",
         "pyyaml",
-        "marshmallow",
+        'marshmallow==2.19.2',

```

Comment:
I think we should allow for a range of versions here, since python-piperci is a library and not an application. Since marshmallow appears to use SemVer, we can specify a range of versions that are allowed here. Something like `marshmallow=>2.19.2,<3`. This way we aren't causing dependency hell on the applications that consume this library by hardcoding exact dependency versions.